tasks.rb

```diff
@@ -1953,6 +1953,9 @@ def apply_hpxml_modification(hpxml_file, hpxml)
       hpxml_bldg.water_heating.water_fixtures_weekday_fractions = '0.012, 0.006, 0.004, 0.005, 0.010, 0.034, 0.078, 0.087, 0.080, 0.067, 0.056, 0.047, 0.040, 0.035, 0.033, 0.031, 0.039, 0.051, 0.060, 0.060, 0.055, 0.048, 0.038, 0.026'
```

Comment:
Why not use `FixturesWeekdayFractions` from `schedules.rb` here?

Reply:
If they are the same, then go for it.